packages/lists/src/components/StaticList/StaticList.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { IStaticListProps } from './StaticList.types';
+
+function defaultRowRenderer<T>(item: T, index: number): JSX.Element {
+  return <li key={index}>{item}</li>;
+}
+
+class StaticList<T> extends React.PureComponent<IStaticListProps<T>> {
```

Comment:
@JasonGore, should I consider making this a functional component? I couldn't figure out how that reads while maintaining the generic type parameter.

Reply:
It's an interesting question. If you want to keep the pure functionality, you would have to use [something like `React.memo` ](https://logrocket.com/blog/pure-functional-components/) to be equivalent to a `PureComponent` class. 

However, @dzearing's investigations have previously found that any pure components with children can't take advantage of pure functionality, and I see `IStaticListProps` here has `children`, so I'm wondering if using pure gets you anything here at all. (I'm not sure if the more specific `children` type you've specified affects this observation.)

Reply:
That's right, the `children` are either an array or object depending on if there are multiple which are always new when re-evaluated thus the container does not benefit from being "pure". So likely not. I'll remove.

I thought it might make sense to make this component functional, but I wasn't sure how to retain the generic type `T`.

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/9537/commits/c7fe79ee1c7dac3338516ade235c943fde856790